Tests/BuildTests/BuildTest.swift

```diff
@@ -10,49 +10,70 @@ import XCTest
         }
 
         let components = #file .split(separator: "/")
         let rootDir = "/" + components [0 ... components.count - 4].joined(separator: "/")
 
-        let examples = ["React", "PINRemoteImage", "Texture", "BasiciOS"]
-        examples.forEach {
-            example in
+        // Give 3 attempts to do the fetch. This is a workaround for flaky
+        // networking
+        let firstFetchResult = (0...3).lazy.compactMap {
+        i -> CommandOutput? in
+            sleep(UInt32(i * 3))
+            print("Starting fetch task", i, example)
             let fetchTask = ShellTask(command: "/bin/bash", arguments: [
                     "-c",
                     "make -C \(rootDir)/Examples/\(example) fetch"
-                    ], timeout: 1200.0)
+                    ], timeout: 1200.0, printOutput: true)
             let fetchResult = fetchTask.launch()
-            XCTAssertEqual(fetchResult.terminationStatus, 0)
-            guard fetchResult.terminationStatus == 0 else {
-                fatalError(
-                        "Can't setup test root."
-                                + "\nCMD:\n\(fetchTask.debugDescription)"
-                                + "\nSTDOUT:\n\(fetchResult.standardOutputAsString)"
-                                + "\nSTDERR:\n\(fetchResult.standardErrorAsString)"
-                )
-            }
-            let bazelScript = "make -C \(rootDir)/Examples/\(example)"
-            print("running bazel:", bazelScript)
-            let buildResult = ShellTask(command: "/bin/bash", arguments: [
-                    "-c", bazelScript ], timeout: 1200.0).launch()
-            XCTAssertEqual(buildResult.terminationStatus, 0, "building \(example)")
-            print("Bazel completed with stderror:", buildResult.standardErrorAsString)
-            print("Bazel completed with stdout:", buildResult.standardOutputAsString)
-
-            // For now, we assume that the if bazel exits with 0 it passed.
-            if buildResult.terminationStatus != 0 {
-                print("Bazel failed with stderror:", buildResult.standardErrorAsString)
-                print("Bazel failed with stdout:", buildResult.standardOutputAsString)
+            if fetchResult.terminationStatus == 0 {
+                return fetchResult
             }
+            return nil
+        }.first
+
+        guard let fetchResult = firstFetchResult else {
+            fatalError("Can't setup test root.")
         }
+        XCTAssertEqual(fetchResult.terminationStatus, 0)
+        let bazelScript = "make -C \(rootDir)/Examples/\(example)"
+        print("running bazel:", bazelScript)
+        let buildResult = ShellTask(command: "/bin/bash", arguments: [
+                "-c", bazelScript ], timeout: 1200.0, printOutput: true).launch()
+        XCTAssertEqual(buildResult.terminationStatus, 0, "building \(example)")
+        print("Bazel completed with stderror:", buildResult.standardErrorAsString)
```

Comment:
This output printing is redundant as we stream standard output and standard error